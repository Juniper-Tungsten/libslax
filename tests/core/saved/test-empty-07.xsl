<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
  <xsl:variable name="v1" select="&quot;before{during}after&quot;"/>
  <xsl:variable name="v2" select="&quot;two&quot;"/>
  <xsl:variable name="e1">
    <e1 a1="{$v1}-{{x}}-{$v2}"/>
  </xsl:variable>
  <xsl:variable name="e2">
    <e2 a1="{substring-before($v1, &quot;{&quot;)}-{$v2}"/>
  </xsl:variable>
  <xsl:template match="/">
    <top>
      <xsl:copy-of select="$e1"/>
      <xsl:copy-of select="$e2"/>
    </top>
  </xsl:template>
</xsl:stylesheet>
