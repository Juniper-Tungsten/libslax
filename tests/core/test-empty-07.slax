version 1.0;

var $v1 = "before{during}after";
var $v2 = "two";

var $e1 = <e1 a1 = $v1 _ "-{x}-" _ $v2>;
var $e2 = <e2 a1 = substring-before($v1, "{") _ "-" _ $v2>;

match / {
    <top> {
	copy-of $e1;
	copy-of $e2;
    }
}
