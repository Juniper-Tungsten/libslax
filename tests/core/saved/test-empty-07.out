<?xml version="1.0"?>
<top>
  <e1 a1="before{during}after-{x}-two"/>
  <e2 a1="before-two"/>
</top>
